### PR TITLE
feat: replace all places where global constants are referred with references to newly generated GlobalConstants class

### PR DESCRIPTION
**Pull request by powerslider**

### CODE REVIEW

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/call/oob/opcodes/DeploymentReturn.java
Comment (lorenzogentile404) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/call/oob/opcodes/DeploymentReturn.java:1]: Does `deployment` replace `return`?

**Review by lorenzogentile404 — COMMENTED** [review]
LGTM, specifically `oob` and `mxp` where I am most familiar. Do you have a list of the substitution you made (e.g., `Trace.oob.EVM_INST_L`T -> `GlobalConstants.EVM_INST_LT`). It would be way easier to review it if you made something like that or you made a little script for substitution.

**Review by letypequividelespoubelles — APPROVED** [review]
LGTM

File: arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/call/oob/opcodes/DeploymentReturn.java
Comment (OlivierBBB) [reply arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/imc/call/oob/opcodes/DeploymentReturn.java:1]: Yes,  `OOB_INST_deployment` the name in the spec, too.

**Pull request merged**